x-pack/test/functional/page_objects/infra_hosts_view.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../ftr_provider_context';
```

Comment:
I am just curious - what does `ftr` stands for here?

Reply:
I think it stands for Functional Test Runner